app/code/Magento/GoogleGtag/Block/Code.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Google AdWords Code block
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GoogleGtag\Block;
+
+use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
+use Magento\GoogleGtag\Helper\GtagConfiguration;
+
+/**
+ * Google Ads Code block
+ *
+ * @api
+ * @since 100.0.2
+ */
+class Code extends Template
```

Comment:
Looking at logic in this block makes me think that it can be completely replaced by a view model making the implementation somewhat cleaner.

Please use this dev docs article as a reference: https://devdocs.magento.com/guides/v2.4/extension-dev-guide/view-models.html